package-lock.json

```diff
@@ -22,6 +22,7 @@
         "custom-gantt-task-react": "^0.4.5",
         "date-fns": "^2.30.0",
         "date-fns-tz": "^2.0.0",
+        "dayjs": "^1.11.6",
```

Comment:
Where is this needed?

Reply:
its a peer dependency for libray date-range-picker
![image](https://github.com/nerubia/kh360-react/assets/148036740/1262137e-f698-46a8-9d18-0d7097b8513f)

Reply:
Ohh I see, thanks! 🚀